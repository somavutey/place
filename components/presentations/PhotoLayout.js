import { Container, ImageList, ImageListItem } from "@mui/material";
import { Box } from "@mui/system";

const PhotoLayout = ({ arrayUrl }) => {
  const urls = arrayUrl?.slice(3);
  return (
    <div style={{ margin: "20px 0" }}>
      <Box
        sx={{
          width: "100%",
          height: 450,
          overflowY: "scroll",
          borderRadius: "50px",
        }}
      >
        <ImageList variant="masonry" cols={2} gap={8}>
          {urls?.map((url, index) => {
            return (
              <ImageListItem key={index}>
                <img src={url} loading="lazy" />
              </ImageListItem>
            );
          })}
        </ImageList>
      </Box>
    </div>
  );
};

export default PhotoLayout;
