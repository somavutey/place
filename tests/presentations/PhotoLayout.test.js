import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import PhotoLayout from "./../../components/presentations/PhotoLayout";

it("PhotoLayout", () => {
  const wrapper = shallow(<PhotoLayout />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
