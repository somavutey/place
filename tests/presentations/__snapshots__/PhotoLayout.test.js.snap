// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhotoLayout 1`] = `
<div
  style={
    Object {
      "margin": "20px 0",
    }
  }
>
  <ForwardRef(Box)
    sx={
      Object {
        "borderRadius": "50px",
        "height": 450,
        "overflowY": "scroll",
        "width": "100%",
      }
    }
  >
    <ForwardRef(ImageList)
      cols={2}
      gap={8}
      variant="masonry"
    />
  </ForwardRef(Box)>
</div>
`;
